src/utils/getUrlForTrustIconFallback.ts

```diff
@@ -9,5 +9,8 @@ export default function getUrlForTrustIconFallback(
   if (type && type !== AssetType.token) {
     network = type;
   }
+  if (type && type === AssetType.bsc) {
```

Comment:
we are keeping like this for now since if we change it we'll have issues merging in new changes from the trust repo